.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "compact": false,
-  "presets": ["env", "stage-2"],
+  "presets": ["env", "stage-2", "react"],
```

Comment:
to avoid unnecessary issues (like try to use react in webpack configs) the `react` preset should not be available outside off test environment or application code. JEST sets NODE_ENV to 'test', so u can setup 'react' preset for only test env here. 
https://babeljs.io/docs/usage/babelrc/#env-option
https://facebook.github.io/jest/docs/en/getting-started.html#additional-configuration